src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -425,6 +439,37 @@ def app_tail_log(cmd, client, resource_group, service, name, instance=None, foll
         raise exceptions[0]
 
 
+def app_identity_assign(cmd, client, resource_group, service, name):
+    app_resource = models.AppResource()
+    identity = models.ManagedIdentityProperties(type="systemassigned")
```

Comment:
do you have plan to support other type? if yes better to expose type as a param with default value

Reply:
Not yet. But we can refactor code if we will support in future.